content/en/docs/languages/cpp/quickstart.md

```diff
@@ -98,13 +98,15 @@ Clone the `grpc` repo and its submodules:
 ```sh
 $ git clone --recurse-submodules -b {{< param grpc_vers.core >}} https://github.com/grpc/grpc
 ```
-#### Build and install gRPC and Protocol Buffers
+#### Build and install gRPC, Protocol Buffers, and Abseil
 
 While not mandatory, gRPC applications usually leverage [Protocol Buffers][pb]
 for service definitions and data serialization, and the example code uses
 [proto3][].
 
-The following commands build and locally install gRPC and Protocol Buffers:
+gRPC uses [Abseil C++ library][abseil] so it needs to be installed as well.
```

Comment:
> gRPC uses <ins>the</ins> [Abseil C++ library][abseil]<ins>,</ins> so it

I'd suggest that you add "the", and a comma after "library".

Reply:
Thanks for the suggestion. Done.